python-selenium/main.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
I'd rather have this under `src/main.py`, so that the templates would overwrite this. It would be confusing if there are both `src/main.py` and `main.py` files in the actors.